src/components/UserTable/UserTable.jsx

```diff
@@ -24,71 +26,127 @@ import AUTH_ROLES from '../../utils/AuthConfig';
-                <EmailIcon mr={2} color="black" />
-                <Text color="black">Email Address</Text>
-              </Flex>
-            </Th>
-            <Th>
-              <Flex align="center">
-                <PhoneIcon mr={2} color="black" />
-                <Text color="black">Phone Number</Text>
-              </Flex>
-            </Th>
-            {isSuperAdmin ? (
+    <>
+      <TableContainer border="1px" borderColor="gray.200">
+        <Table variant="striped" colorScheme="gray">
+          <Thead bgColor="rgb(247,250,252)">
+            <Tr className={styles['table-header']}>
+              <Th>
+                <Flex align="center">
+                  <Image src={peopleIcon} mr={2} color="black" />
+                  <Text color="black">Name</Text>
+                </Flex>
+              </Th>
+              <Th>
+                <Flex align="center">
+                  <EmailIcon mr={2} color="black" />
+                  <Text color="black">Email Address</Text>
+                </Flex>
+              </Th>
               <Th>
-                <Flex>
-                  <Image src={cardAccount} mr={2} />
-                  <Text color="black">Role</Text>
+                <Flex align="center">
+                  <PhoneIcon mr={2} color="black" />
+                  <Text color="black">Phone Number</Text>
                 </Flex>
               </Th>
-            ) : null}
-            <Th />
-          </Tr>
-        </Thead>
-        <Tbody className={styles['row-text']}>
-          {users.map(user => (
-            <Tr key={`${user.id}`}>
-              <Td>
-                {user.firstName} {user.lastName}
-              </Td>
-              <Td>{user.email}</Td>
-              <Td>{formatPhone(user.phoneNumber)}</Td>
               {isSuperAdmin ? (
-                <Td>
-                  {user.role === ADMIN_ROLE || user.role === SUPERADMIN_ROLE ? (
-                    <Tag size="sm" variant="solid" colorScheme="blue" font="Inter" fontSize="14px">
-                      Admin
-                    </Tag>
-                  ) : (
-                    <Tag size="sm" variant="solid" colorScheme="red" font="Inter" fontSize="14px">
-                      Driver
-                    </Tag>
-                  )}
-                </Td>
+                <Th>
+                  <Flex>
+                    <Image src={cardAccount} mr={2} />
+                    <Text color="black">Role</Text>
+                  </Flex>
+                </Th>
               ) : null}
-              <Td>
-                <EditAccountModal mr={5} isSuperAdmin={isSuperAdmin} staffProfile={user} />
-                {isSuperAdmin ? <DeleteAccountModal /> : null}
-              </Td>
+              <Th />
             </Tr>
-          ))}
-        </Tbody>
-      </Table>
-    </TableContainer>
+          </Thead>
+          <Tbody className={styles['row-text']}>
+            {users.map(user => (
+              <Tr key={`${user.id}`}>
```

Comment:
Use `user.email` instead of `user.id` since we get PropType errors when adding a new user locally (since they haven't been assigned an `id` yet)